src/powershell/install-boxstarter.ps1

```diff
@@ -0,0 +1,43 @@
+# Install and configure Boxstarter
+# See: https://boxstarter.org/
+function Install-Boxstarter {
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://boxstarter.org/bootstrapper.ps1'))
+    Get-Boxstarter -Force
+
+    # Fix verbosity issues with Boxstarter v3
+    # See: https://github.com/chocolatey/boxstarter/issues/501
```

Comment:
This issue was closed as resolve in the [3.0.1 release](https://github.com/chocolatey/boxstarter/releases/tag/3.0.1). Is there additional context for what you're doing here and why? I don't see any of this in the GitHub issue you link here.

Reply:
@mcdonnnj resolved in [dd989b3](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/dd989b3a9d9435e0e05bdfb2931bc46206bf1a84)